src/ess/amor/load.py

```diff
@@ -33,43 +33,112 @@ def load_events(detector: RawDetector[Run]) -> RawEvents[Run]:
     return RawEvents[Run](data)
 
 
+class _Detector:
+    nBlades = sc.scalar(14)  # number of active blades in the detector
+    nWires = sc.scalar(32)  # number of wires per blade
+    nStripes = sc.scalar(64)  # number of stipes per blade
+    angle = sc.scalar(5.1, unit='degree').to(
+        unit='rad'
+    )  # deg  angle of incidence of the beam on the blades (def: 5.1)
+    dZ = sc.scalar(4.0, unit='mm') * sc.sin(
+        angle
+    )  # mm  height-distance of neighboring pixels on one blade
+    dX = sc.scalar(4.0, unit='mm') * sc.cos(
+        angle
+    )  # mm  depth-distance of neighboring pixels on one blace
+    bladeZ = sc.scalar(10.455, unit='mm')  # mm  distance between detector blades
+    zero = 0.5 * nBlades.value * bladeZ  # mm  vertical center of the detector
+    distance = sc.scalar(
+        4000, unit='mm'
+    )  # mm  distance from focal point to leading blade edge
```

Comment:
Should this be a `dataclass`? Also, use docstrings instead of comments?

Reply:
I don't think it needs to be a dataclass. It's a namespace for constants, not a class that is meant to be instantiated.